src/layouts/components/Homepage/HeroBody.tsx

```diff
@@ -307,45 +307,52 @@ const HeroLayoutForm = ({
   })
 
   return (
-    <>
+    <VStack spacing="1rem" align="flex-start" w="100%" ref={ref}>
       {showNewLayouts && inView && (
-        <FeatureTourHandler
-          localStorageKey={LOCAL_STORAGE_KEYS.HeroOptionsFeatureTour}
-          steps={HERO_OPTIONS_FEATURE_STEPS}
-        />
+        /**
+         * Because of the VStack, this renders as a valid child with padding
+         * which messes up the layout. The caller of this method also wraps
+         * this around a VStack, so placing the FeatureTourHandler outside of this
+         * parent VStack does not solve the issue. As a workaround, the margin
+         * top is set to -1 to 'undo' the spacing by VStack.
+         */
+        <Box mt="-1rem">
```

Comment:
Yes this is ugly, but I am not sure how to quickly fix this without modifying a huge chunk of the code. 
### Before
<img width="415" alt="Screenshot 2023-09-22 at 3 19 55 PM" src="https://github.com/isomerpages/isomercms-frontend/assets/42832651/2024ac21-203c-431d-aea1-9552cce60810">

### After
<img width="422" alt="Screenshot 2023-09-22 at 3 19 36 PM" src="https://github.com/isomerpages/isomercms-frontend/assets/42832651/e84a1eaf-b62c-4b45-af2c-5a955f95db3e">